recipes/barrnap-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = 'barrnap' %}
+{% set version = '0.0.5' %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "acac783c1f93e8e9585017be9e128f7f95d8ae77a9ed3b3656fa53d5043e0e21" %}
+
+package:
+  name: barrnap-python
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
This should be one line. # [win or py2k or py3k]

Reply:
The win skip is not needed and I'm wondering if the py34 skip is really needed.

Reply:
Thanks, I changed that.  The pertinent change between py3.4 and later versions is the syntax for running system commands (subprocess.call() vs the more updated subprocess.run())